CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+## Contribution guidelines
+
+Contributions to `becquerel` are welcome and encouraged, whether it is
+reporting bugs, requesting features, or contributing code.
+Please follow these guidelines when contributing to this project.
+
+### Reporting issues
+
+When reporting issues with `becquerel`, please provide a minimum working example to help identify the problem and tag the issue as a `bug`.
+
+### Feature requests
+
+For a feature request, please create an issue and label it as a `new feature`.
+
+### Checklist for code contributions:
+  - [ ] Branch off of `develop` and name the branch `feature-XX` or `issue-XX`
+  - [ ] Develop the feature or fix
+  - [ ] Write tests to cover all use cases
+  - [ ] Ensure test coverage is >95%
+  - [ ] Ensure all tests pass (`python setup.py test`)
+  - [ ] Ensure there are no linter issues (both `flake8` and `pylint`)
+  - [ ] Check style is [consistent with Google Python style guide](https://google.github.io/styleguide/pyguide.html)
```

Comment:
Just realized that the style guide is deprecated, we probably should also move to https://github.com/google/styleguide/blob/gh-pages/pyguide.md